src/Constants.js

```diff
@@ -1,10 +1,14 @@
 const RequestType = {
-  PasswordReset: 'passwordReset',
-  Cancelled: 'cancelled',
-  Other: 'other',
-  Code: 'code',
-  Logout: 'logout',
-  Ignore: 'ignore',
+  PasswordReset: "passwordReset",
+  Cancelled: "cancelled",
+  Other: "other",
+  Code: "code",
+  Logout: "logout",
+  Ignore: "ignore"
 };
 
-export { RequestType };
+const azureErrors = {
+  wrongEndpoint: "AADB2C90088"
```

Comment:
Add reference to microsoft documentation for easy look-up for future developers?